sunpy/timeseries/sources/eve.py

```diff
@@ -57,32 +57,48 @@ class ESPTimeSeries(GenericTimeSeries):
 
     _source = 'esp'
 
-    @peek_show
-    def peek(self, title='EVE/ESP Level1', **kwargs):
+    def plot(self, title="EVE/ESP Level 1", **kwargs):
         """
+        Plots the EVE ESP Level 1 timeseries data.
+
         Parameters
         ----------
         title : `str`, optional
-            Plot title.
+            The title of the plot. Defaults to "EVE/ESP Level 1".
         **kwargs : `dict`
             Additional plot keyword arguments that are handed to `pandas.DataFrame.plot`.
-        """
 
+        Returns
+        -------
+        `~matplotlib.axes.Axes`
```

Comment:
This actually returns an array of `Axes`.
```suggestion
        array of `~matplotlib.axes.Axes`
```